website/static/js/nodeControl.js

```diff
@@ -46,6 +46,27 @@ var ProjectViewModel = function(data) {
     self.user = data.user;
     self.nodeIsPublic = data.node.is_public;
     self.nodeType = data.node.node_type;
+    self.instLogoPath = ko.observable('');
+
+    self.fetchInstitutions = function() {
+        if (data.node.institution) {
+            var url = window.contextVars.apiV2Prefix + 'nodes/' + self._id + '/institution/';
+            return $osf.ajaxJSON(
+                'GET',
+                url,
+                {isCors: true}
+            ).done(function (response) {
+                self.instLogoPath(response.data.attributes.logo_path);
```

Comment:
Rather than sending an extra request just to get the logo path, could you add it to the `data` payload that is passed to the constructor. Yes, this will probably mean modifying v1 code, but I think it's justified in this case. By saving a request, we should reduce the "flashing" of the logo to some degree.